shell/imports/blackrock-payments/server/payments-api-server.js

```diff
@@ -251,7 +252,7 @@ BlackrockPayments.registerPaymentsApi =
         if (user) {
           sendInvoice(this._db, user, this._invoice, this._config);
         } else {
-          console.error("Stripe charge didn't match any user: " + chrage.id);
+          console.error("Stripe charge didn't match any user: " + charge.id);
```

Comment:
This was a bug, but a pretty immaterial one that would only get hit if we got a spurious Charge object from Stripe, which would be quite unusual.